feature/experimental/lacp_and_base_interface/singleton_interface/feature.textproto

```diff
@@ -152,3 +152,40 @@ telemetry_path {
 telemetry_path {
   path: "/interfaces/interface/subinterfaces/subinterface/ipv6/state/counters/out-pkts"
 }
+telemetry_path {
```

Comment:
We should reorder these to keep the config and telemetry paths close and in this order.